script-templates/Splunk.OTel.DotNet.psm1.template

```diff
@@ -241,7 +241,7 @@ function Install-OpenTelemetryCore() {
         [string]$LocalPath
     )
 
-    $version = "v1.0.2"
+    $version = "vVERSION_PLACEHOLDER"
```

Comment:
Can we change `VERSION_PLACEHOLDER` to e.g. `{{ VERSION_PLACEHOLDER }}` to make it visually easier to read?

Reply:
`{{ }}` syntax should be OK. See: https://github.com/scriban/scriban